src/random.ts

```diff
@@ -98,15 +98,17 @@ export class Random {
    * Returns random element from the given array.
    *
    * @template T The type of the entries to pick from.
-   * @param array Array to pick the value from. Defaults to `['a', 'b', 'c']`.
+   * @param array Array to pick the value from.
    *
    * @example
-   * faker.random.arrayElement() // 'b'
    * faker.random.arrayElement(['cat', 'dog', 'mouse']) // 'dog'
+   * faker.random.arrayElement([0, 1, 2, 3]) // 3
    */
-  arrayElement<T = string>(
-    array: ReadonlyArray<T> = ['a', 'b', 'c'] as unknown as ReadonlyArray<T>
-  ): T {
+  arrayElement<T>(array: ReadonlyArray<T>): T {
+    if (array.length <= 0) {
```

Comment:
This needs to handle undefined arrays as well. When is the array length below zero?

Maybe like this?

```suggestion
    if (typeof array === 'undefined' || !array.length) {
```

Reply:
I would even go with `array == null || array.length === 0`

Reply:
Handled both `null` and `undefined` in f9599b6aa35cac02f2dd9e859ae930218bc1dbd1